web/filter/jwt/jwt_test.go

```diff
@@ -78,12 +78,12 @@ func TestVerifyJWT_no_JWT(t *testing.T) {
 
 	name, err := GetName(r.Context())
 	require.Equal(t, "", name)
-	require.EqualError(t, err, "failed to get name: no user in context")
-	require.EqualError(t, errors.Unwrap(err), "no user in context")
-	isAdmin, err := IsAdmin(r.Context())
+	require.EqualError(t, err, "failed to get name: failed to get token claims: no token in context")
+	require.EqualError(t, errors.Unwrap(err), "no token in context")
+	isAdmin, err := HasRole(r.Context(), "staff")
```

Comment:
Having gotten rid of the `admin` term in function names, we should consider simply dropping it entirely and going for something more neutral in the tests for this otherwise generic package (e.g. `hasRole` instead of `isAdmin` and `valid_JWT_with_role` and `valid_JWT_without_role` instead of `valid_JWT_is_admin` and `valid_JWT_is_not_admin`).

Reply:
fixed (at least in the places where I changed the tokens to include the "staff" role). This improves documentation value of the tests.